wire-ios-testing/Source/Public/ZMTBaseTest.swift

```diff
@@ -21,10 +21,9 @@ import Foundation
 extension ZMTBaseTest {
     @objc
     public static func checkForMemoryLeaksAfterTestClassCompletes() {
-        if MemoryReferenceDebugger.aliveObjects.count > 0 {
-            print("Leaked: \(MemoryReferenceDebugger.aliveObjectsDescription)")
-            assert(false)
-        }
+        let leakedObjects = "Leaked: \(MemoryReferenceDebugger.aliveObjectsDescription)"
+        print(leakedObjects)
+        XCTAssert(MemoryReferenceDebugger.aliveObjects.isEmpty, leakedObjects)
```

Comment:
Looks good, I ask myself if this really fails on tests when memory leaks occur or the error is now invisible?

Reply:
Yes the results her contained the failed test:
<img width="730" alt="Screenshot 2024-03-28 at 13 00 46" src="https://github.com/wireapp/wire-ios/assets/7196958/a242c123-3ed3-44ce-bfc9-7c7b15b7e04c">

Reply:
https://github.com/wireapp/wire-ios/actions/runs/8466550736/job/23195584089?pr=1190